getting-started/scaling-out.md

```diff
@@ -86,11 +86,14 @@ foreign data wrapper and any data node (server) objects.
 Deleting a data node is done by calling `delete_data_node`:
 
 ```sql
-SELECT delete_data_node('node1');
+SELECT delete_data_node('node1', cascade=>'true');
 ```
 >:TIP: Note that a data node cannot be deleted if it contains data for a
 hypertable, since otherwise data would be lost.
 
+>:WARNING: Although the `cascade` parameter is not strictly a required argument,
+you *MUST* set it to `true` to avoid breaking necessary user mappings.
```

Comment:
Per Brian's comment, I was going to remove the last portion and just go with >:WARNING: Although the `cascade` parameter is not strictly a required argument, you *MUST* set it to `true`?

So perhaps update that here too so that they match?

Reply:
I will sync with the actual change if it will be in https://github.com/timescale/docs.timescale.com-content/pull/203

Reply:
Actually, I kind of like perhaps : you MUST set it to true in this current release to avoid putting TimescaleDB in an error state?

Reply:
Synced